framework/routing/__init__.py

```diff
@@ -392,12 +413,15 @@ def __init__(self, template_name, renderer=None, error_renderer=None,
                      to add to data from view function
         :param detect_render_nested: Auto-detect renderers for nested
             templates?
+        :param safe: Boolean: turn on markup-safe escaping?
```

Comment:
Interesting comment on ambiguous meaning; in this context I had intended it to mean "apply markupsafe filters". I think this meaning may be closer to how the Mako documentation uses the word safe: http://docs.makotemplates.org/en/latest/filtering.html

`h : HTML escaping, provided by markupsafe.escape(string)`

As opposed to turning it off, which is 
`n : disable all default filtering; only filters specified in the local expression tag will be applied.`

Though the Jinja2 documentation uses the word safe in an opposite manner, as you note. I'm not sure what a neutral term would be...?